transfer/message/shape/transfer-request-message-shape.ttl

```diff
@@ -66,7 +68,7 @@ shapes:TransferRequestMessageShape
 		sh:severity sh:Violation ;
 		sh:maxCount 1 ;
 		sh:minCount 1 ;
-		sh:message "<https://raw.githubusercontent.com/International-Data-Spaces-Association/ids-specification/master/schemas/transfer-request-message-shape.ttl> (TransferRequestMessageShape): An dspace:callbackAddress property must point exactly one xsd:anyURI field."@en ;
+		sh:message "<https://raw.githubusercontent.com/International-Data-Spaces-Association/ids-specification/master/schemas/transfer-request-message-shape.ttl> (TransferRequestMessageShape): An dspace:callbackAddress property must point exactly one xsd:string field."@en ;
```

Comment:
Let's stick to "anyURI" but please add this line to the context.json:
`"dspace:callbackAddress": { "@type": "xsd:anyURI" },`